Package.swift

```diff
@@ -15,7 +15,7 @@ let package = Package(
     name: "SpeziFirebase",
     defaultLocalization: "en",
     platforms: [
-        .iOS(.v16)
+        .iOS(.v17)
```

Comment:
Did I miss something or is there no way to specify a target platform of "16.4"? This would currently be the lowest required as we use the new [AuthenticationController](https://developer.apple.com/documentation/authenticationservices/authorizationcontroller)

Reply:
resolved as of the meeting